### PR TITLE
Support for Symfony 4.1

### PR DESCRIPTION
**Pull request by kleinkoerkamp**
Added support for Symfony 4.1. Issue #143

### CODE REVIEW

**Comment by stefandoorn** [comment]
I think we should also start testing this library against multiple Symfony versions and multiple Sylius versions, e.g. like done here: https://github.com/stefandoorn/sitemap-plugin/blob/master/.travis.yml#L7 & https://github.com/stefandoorn/sitemap-plugin/blob/master/.travis.yml#L21. Would you be able to add this @kleinkoerkamp ?

**Comment by stefandoorn** [comment]
Also, do all services need to become public? Or only some (e.g. controllers)? It feels a bit much to me to make all of them public at once.

**Comment by kleinkoerkamp** [comment]
@stefandoorn good point, i will look into that

File: src/DependencyInjection/Compiler/RegisterExporterPass.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
     public function process(ContainerBuilder $container)
     {
         $serviceId = 'sylius.exporters_registry';
-        if (!$container->has($serviceId)) {
+        if ($container->has($serviceId) == false) {
```
Comment (stefandoorn) [inline src/DependencyInjection/Compiler/RegisterExporterPass.php]: Nitpicking, but this can be `===` I think.

Comment (kleinkoerkamp) [reply src/DependencyInjection/Compiler/RegisterExporterPass.php]: `===` will give the following issue:
```
 ------ ----------------------------------------------------------------------------------- 
  Line   src/DependencyInjection/Compiler/RegisterExporterPass.php                          
 ------ ----------------------------------------------------------------------------------- 
  26     Strict comparison using === between true and false will always evaluate to false.  
 ------ ----------------------------------------------------------------------------------- 
```

`!$container->has($serviceId)` will give the following issue.
```
 ------ ----------------------------------------------------------- 
  Line   src/DependencyInjection/Compiler/RegisterExporterPass.php  
 ------ ----------------------------------------------------------- 
  26     Negated boolean is always false.                           
 ------ ----------------------------------------------------------- 
```

Comment (stefandoorn) [reply src/DependencyInjection/Compiler/RegisterExporterPass.php]: Hm :) Ok, let's leave it for now :) Can be adjusted in another PR.

**Comment by dnna** [comment]
Ping :) 
What is the status of this PR? Would really love to have this merged for my project.
If it needs further changes I'd be happy to help.

**Pull request merged**